torch/csrc/jit/passes/onnx/eval_peephole.cpp

```diff
@@ -121,6 +121,7 @@ static void fuseConvBatchNorm(Block* b, ValueToParamPairMap& valsToParamsMap) {
       newConv->copyAttributes(*oldConv);
       newConv->insertBefore(bnNode);
       newConv->addInput(oldConv->inputs().at(0));
+      newConv->copyMetadata(bnNode);
```

Comment:
Why don't we copy these metadata from oldConv node?

Reply:
yes `oldConv` is indeed better.